docs/index.rst

```diff
@@ -1,159 +1,325 @@
 Testimony
 =========
 
-An approach to document test cases in the Python source code
-------------------------------------------------------------
+.. contents:: Topics
 
-Are you tired of managing your test cases in a test case management tool and
-your test code in a Python automation framework?  Testimony can help you to use
-your Python automation framework as a test case repository tool.
+.. _what_is_testimony:
```

Comment:
I think this is breaking the menu check betelgeuse docs [1] there is a way to expand `Usage Examples` and the links are listed also below that item. Since I don't know much about RST I can't say how it is supposed to work. Please let me know if I am missing something here.

[1] http://betelgeuse.readthedocs.io/en/latest/#test-results-command